src/Pages/Home/Home.css

```diff
@@ -0,0 +1,134 @@
+.header {
+  color: white;
+  line-height: 1.2;
+}
+
+.container {
+  height: 100vh;
+  gap: 2rem;
+}
+
+.todo-body {
+  background-color: white;
+  flex: 1 1 auto;
+  border-radius: 30px 30px 0px 0px;
+}
+
+.todo-header {
+  padding: 0 5rem;
+  /* line-height: 1.2; */
+  align-items: center;
+}
+
+.add-btn {
+  color: var(--theme-color);
+  cursor: pointer;
+}
+
+.todo-container {
+  padding: 0 7rem;
+  gap: 1rem;
+}
+
+.todo-item {
+  height: 60px;
+  gap: 1rem;
+  padding: 1rem;
+}
+
+.todo-item .circle {
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  border: 1px solid var(--theme-color);
+  cursor: pointer;
+  flex-shrink: 0;
+}
+
+.todo-text {
+  height: 100%;
+  margin-right: auto;
+  overflow-y: scroll;
+  text-overflow: ellipsis;
+}
+
+.todo-text::-webkit-scrollbar {
```

Comment:
Why is scrollbar being hidden?

Reply:
Made scrollbar visible with custom colour and style

Reply:
Great